docs/user-settings.md

```diff
@@ -20,25 +20,73 @@ Luigi allows you to display a user settings dialog and to manage user data, thro
 Each `userSettingGroup` could have the following meta data:
 
 ```javascript
-userSettingGroup: {
-  label: 'Label',
-  sublabel: 'Sublabel',
-  icon: 'account',
-  title: 'Title',
-  settings: {
-    inputField: { type: 'string', label: 'label' , isEditable: true},
-    checkbox: { type: 'boolean', label: 'Checkbox', isEditable: true },
-    enum:
-      {
-        type: 'enum',
-        label: 'Label',
-        options: ['option1', 'option2', 'option3', 'option3'],
-        description: 'Description'
+settings:{
+...
+},
+navigation:{
+...
+},
+userSettings:{
+  userSettingGroups:{
+    account: {
+      label: 'Account',
+      sublabel: 'account',
+      icon: 'account',
+      title: 'Account Settings',
+      settings: {
+        name: { type: 'string', label: 'Name' , isEditable: true},
+        checkbox: { type: 'boolean', label: 'Checkbox', isEditable: true },
+        enum:
+          {
+            type: 'enum',
+            label: 'Label',
+            options: ['option1', 'option2', 'option3', 'option3'],
+            description: 'Description'
+          },
+          enum2:
+          {
+            type: 'enum',
+            label: 'Label',
+            options: ['option1', 'option2'],
+            style: 'button',
+            description: 'Description'
+          }
       }
+    }
   }
 }
 ```
 
+#### label
+- **type** string (optional)
+- **description** defines the label for the left-side navigation entry.
+#### sublabel
+- **type** string (optional)
+- **description** defines the sublabel for the left-side navigation entry.
+#### icon
+- **type** string (optional)
+- **description** name of the icon, without the `sap-icon--` prefix or path to an image.
+#### title
+- **type** string (optional)
+- **description** title of the user settings group. It will be displayed as a header in the editor area.
+#### viewURL
+- **type** string (optional)
+- **description** points to a custom micro frontend. It is possible to not use Luigi`s user settings editor instead you can write an own editor micro frontend.
+In that case the micro frontend will be displayed in the editor area.
+#### settings
+- **type** object (optional)
+- **description** has objects of settings for the corresponding user group. 
+`Key` of each setting object will be the key in the stored user settings with the corresponding value.
+Following attributes to define a setting objects are:
+- **attributes**
+  - **type** (mandatory) is a string and defines the data type of this setting. It could be `string`, `boolean` or `enum`. If data type `string` is defined, an input field will be rendered in the editor area. If this property is set to `boolean` a switcher will be rendered. If `enum` type is set it will be rendered a dropdown by default.
```

Comment:
```suggestion
  - **type** (mandatory) is a string and defines the data type of this setting. It could be `string`, `boolean` or `enum`. If data type `string` is defined, an input field will be rendered in the editor area. If this property is set to `boolean`, a switcher will be rendered. If `enum` type is set, it will be rendered as a dropdown by default.
```